go.mod

```diff
@@ -9,4 +9,6 @@ require (
 	github.com/magiconair/properties v1.8.1
 	github.com/rs/zerolog v1.15.0
 	github.com/stretchr/testify v1.3.0
+	github.com/urfave/cli/v2 v2.1.1
+	github.com/xmonader/tcprouter v0.0.0-20200117135645-c8b441f277a1 // indirect
```

Comment:
Can we updated this one too to threefoldtech

Reply:
done